jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/Vessel.java

```diff
@@ -47,6 +48,10 @@ public class Vessel {
   @Column(name = "width", columnDefinition = "numeric")
   private Float width;
 
+  @Enumerated(EnumType.STRING)
+  @Column(name = "type")
+  private VesselTypeCode type;
```

Comment:
Either the field should be typeCode or the enum should be VesselType. Having "code" in one but not the other is inconsistent

Reply:
Good point. Renamed to VesselType.